src/Loading/babylon.sceneLoader.ts

```diff
@@ -27,6 +27,7 @@
         load: (scene: Scene, data: string, rootUrl: string, onError?: (message: string, exception?: any) => void) => boolean;

```

Comment:
I plan on removing the importMesh and load methods and replace any existing code that references them with calls to the new loadAssets

Reply:
this will be a breaking change but  don't think it will really be impactful
We will need to document it to make sure we are not hurting existing code too much

Reply:
ok, I didn't remove them but currently any external existing loaders will need to be updated to add the loadAssets method.

Reply:
We should just add a warning if the plugin does not provide loadAssets nor loadAssetsAsync

Reply:
Added a warning when it doesnt exist